.env.EXAMPLE

```diff
@@ -0,0 +1,5 @@
+DISCORD_TOKEN="blablabla"
```

Comment:
Nice suggestion of adding this file, might be interesting to reference it in the readme so people won't miss it as it could be pretty helpful :D